src/ProgressiveHedging.jl

```diff
@@ -147,6 +147,15 @@ Solve the stochastic programming problem described by `tree` and the models crea
 * `worker_assignments::Dict{Int,Set{ScenarioID}}` : Dictionary specifying which scenario subproblems a worker will create and solve. The key values are worker ids as given by Distributed (see `Distributed.workers()`). The user is responsible for ensuring the specified workers exist and that every scenario is assigned to a worker. If no dictionary is given, scenarios are assigned to workers in round robin fashion. Defaults to empty dictionary.
 * `args::Tuple` : Tuple of arguments to pass to `model_cosntructor`. Defaults to (). See also `other_args` and `kwargs`.
 * `kwargs` : Any keyword arguments not specified here that need to be passed to `subproblem_constructor`.  See also `other_args` and `args`.
+
+**Return Arguments**
+
+* `niter` : Number of iterations performed.
+* `abs_res` : Absolute residual (of what?).
```

Comment:
The residual is a combination of two things: the first is the squared L^2 norm of movement between the consensus variables from one iteration to the next. The second is the squared L^2 norm of the lack of consensus.  That is, the sum of squares of the differences of the the scenario specific value and the consensus value.  The absolute residual is the square root of the sum of these two quantities divided by the square root of the number of consensus variables to give the whole thing a kind of sample standard deviation look.  The relative residual is then the absolute residual divided by the maximum consensus variable value.  The code for the computation can all be found in the algorithm.jl file.